tests/test_default.py

```diff
@@ -0,0 +1,21 @@
+def test_brew_taps(Command):
+    c = Command('/usr/local/bin/brew tap --list')
+    assert c.rc == 0
+    tapped = c.stdout.split()
+    assert 'homebrew/core' in tapped
```

Comment:
Sure, I'll split up the changes into separate logical PRs. It started smaller but gradually got away from me...

Perhaps I misspoke. The "developers" in those scenarios were meant to be potential contributors to _this_ project. I agree that users should not be attempting to modify the code for this role in any other context. What I see happening is that the Travis build passes, and only code-review stands in the way of introducing potentially harmful/breaking changes. 

In fact, that speaks pretty well to what I'm going to say next...

What I take away from your last response is that you have concerns about the maintainability of these changes. I certainly appreciate how challenging it can be to manage that many projects. However, I will assert (to the continued annoyance of my friends/coworkers) that contract-defining tests, even for infrastructure automation, actually makes projects easier to maintain. The cost of writing/maintaining those tests is vastly outweighed by the benefit of having them. There is no doubt a longer a conversation we could have on that topic (happy to take it offline). 

Let me know if you'd like any assistance with maintaining this project. Feel free to add me as a collaborator! 😄 

FWIW, the `molecule` portion makes local testing _really_ easy.